_posts/2021-11-03-flink-backward.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Flink Backward - The Apache Flink Retrospective"
+date: 2021-11-03 00:00:00
+authors:
+- joemoe:
+  name: "Johannes Moser"
+excerpt: A look back at the development cycle for Flink 1.14 
+---
+
+It has now been a month since [Apache Flink 1.14](https://flink.apache.org/downloads.html#apache-flink-1140) has been released into the wild. :partying_face: We had a comprehensive look at the enhancements, additions, and fixups in the [release announcement blog post](https://flink.apache.org/news/2021/09/29/release-1.14.0.html) and now we will have a look at the development cycle from a different angle. Based on feedback collected from contributors involved in this release, we will explore the experiences and processes behind it all.  
+
+{% toc %}
+
+# A retrospective on the release cycle
+
+From the team, we collected emotions that have been attributed to points in time of the 1.14 release cycle:
+
+<center>
+<img src="{{site.baseurl}}/img/blog/2021-11-03-flink-backward/1.14-weather.png" width="70%"/>
+</center>
+
+The overall sentiment seems to be quite good. A ship crushed a robot two times, someone felt sick towards the end, an octopus causing negative emotions appeared in June...
+
+We looked at the origin of these emotions and came up with an analysis of what went well and what could be improved. We also incorporated some feedback gathered from the community.
+
+## Problems faced
+
+From a content perspective, the community is still ironing out processes around documentation and blog posts. There have been a lot of test instabilities and issues with some of the efforts to fix them. We also had to push the feature freeze by two weeks, which might actually be considered early and not actually be living up to the Apache Flink release tradition. :p
+
```

Comment:
This feels like 3 unrelated sentences smashed together. How about something like this:

```suggestion
No release is perfect, and the community is constantly looking to improve.  

Apache Flink has active contributors from around the globe, many of whom do not speak English as a first language. The community is still ironing out processes for delivering high-quality documentation and blog posts from a content perspective. We have added contribution guidelines for writing and have contributors focusing on this component. 

Each Flink release is built with the help of hundreds of contributors, each working on different parts of the project. Changes to one module may affect others in ways that are not always obvious. To maintain quality, the community supports an expansive test suite. Invariably some tests are found to be flakey. Whenever we discover a test issue, the community opens a blocker issue that we must resolve before the next release. In practice, this leads to contributors triaging most test instabilities towards the end of each release cycle. From now on, we want to be more mindful of these failures and prioritize them when discovered. 

Finally, the community pushed the planned feature freeze for 1.14 by two weeks. Two weeks is an improvement from previous release cycles, but we hope to continue improving this metric 1.15. 
```

Reply:
I know see I've added some overlap to what's written below, so happy to change this but I think it needs more than what was already there.